src/markdown-converter.js

```diff
@@ -0,0 +1,15 @@
+import Markdown from 'markdown-it';
+
+const converter = new Markdown('zero', {
+  breaks: true,
+  typographer: true,
+});
+
+converter
+  .enable('backticks')
+  .enable('emphasis')
+  .enable('link')
+  .enable('list')
+  .enable('smartquotes');
+
+export default converter;
```

Comment:
This file doesn't have tests, not sure what I should test really... examples?

Reply:
nah. seems good to me like it is. we would just test markdown-it